src/Pkg.jl

```diff
@@ -97,6 +97,18 @@ See also [`PackageSpec`](@ref).
 """
 const add = API.add
 
+"""
+    Pkg.precompile()
+
+Precompile all the dependencies of the project by running `import` on all of them in a new process.
```

Comment:
```suggestion
Precompile all the dependencies of the project.
```

It does not run `import` on them in a new process anymore.

Reply:
Done.